src/workerd/api/node/util.c++

```diff
@@ -3,8 +3,6 @@
 //     https://opensource.org/licenses/Apache-2.0
 #include "util.h"
 
-#include <workerd/jsg/jsg.h>
```

Comment:
Some of these includes should be kept based on IWYU – jsg.h is being used here. https://include-what-you-use.org/

Reply:
`#include "util.h"` already includes them. should we still include them here?

Reply:
Yes https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/WhyIWYU.md